react/testing_5/src/components/Field/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import './styles.css';
+
+const Field = ({
+  input,
+  label,
+  type,
+  meta: { touched, error, warning },
+  syncError
+}) => (
+  <div className="field-container">
+    <label>{label}</label>
+    <div>
+      <input {...input} placeholder={label} type={type} />
+      {touched && error && <span className={`error error-${input.name}`}>{error}</span>}
```

Comment:
These classNames aren't declared at `Field/styles.css` 🤔

Reply:
They are for selection purposes. If this was a real project, these errors would have styles.